### PR TITLE
Added seeds and dependent destroy to relevant models

### PR DESCRIPTION
**Pull request by florestankorp**

### CODE REVIEW

File: app/models/teacher.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ class Teacher < ApplicationRecord
   attr_accessor :name
 
   belongs_to :user
-  has_many :evaluations
+  has_many :evaluations, dependent: :destroy
```
Comment (arnoFleming) [inline app/models/teacher.rb]: nope

File: app/models/student.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 class Student < ApplicationRecord
-  has_many :evaluations
+  has_many :evaluations, dependent: :destroy
```
Comment (arnoFleming) [inline app/models/student.rb]: yes

File: db/seeds.rb [hunk 3]
```diff
@@ -7,11 +7,42 @@
...
+    user_id: User.create!(
+      email: Faker::Internet.email,
+      password: Faker::Internet.password,
+    ).id,
+  )
+end
+
+puts "created 25 students"
+
+Student.all.each do |students|
```
Comment (arnoFleming) [inline db/seeds.rb]: rename `students` to `student`, since it is just 1 student

Comment (arnoFleming) [inline db/seeds.rb]: You could even create the evaluations for the student in the loop above (but now I'm nitpicking)

**Comment by arnoFleming** [comment]
Allmost there!

File: db/seeds.rb [hunk 4]
```diff
@@ -7,11 +7,42 @@
...
+    user_id: User.create!(
+      email: Faker::Internet.email,
+      password: Faker::Internet.password,
+    ).id,
+  )
+end
+
+puts "created 25 students"
+
+Student.all.each do |student|
```
Comment (arnoFleming) [inline db/seeds.rb]: Unused block argument - `student`. You can omit the argument if you don't care about it.

Comment (arnoFleming) [inline db/seeds.rb]: Dan moet je de `students` een paar regels hieronder wel OOK aanpassen 😁

**Comment by florestankorp** [comment]
done

File: db/seeds.rb (hunk 4 above)
Comment (florestankorp) [inline db/seeds.rb]: done

**Pull request merged**